src/v/cluster/cluster_discovery.cc

```diff
@@ -69,7 +69,7 @@ std::vector<model::broker> cluster_discovery::initial_seed_brokers() const {
     // just this node as the initial seed.
     if (config::node().empty_seed_starts_cluster()) {
         if (config::node().seed_servers().empty())
-        return {make_self_broker(config::node())};
+            return {make_self_broker(config::node())};
```

Comment:
unrelated change

Reply:
that's a conflict resolution artifact, will fix it